tracing-example/src/main/java/org/jboss/resteasy/tracing/examples/jsonb/Play2.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.resteasy.tracing.examples.jsonb;
+
+import java.lang.reflect.Method;
+
+public class Play2 {
```

Comment:
What's this class for? I don't see it mentioned in the doc nor in the code...

Reply:
removed